### ISSUE TITLE
Cargo warning

### ISSUE THREAD

**Issue by balroggg** [Xithrius/twitch-tui#132]
Warning when build project
```
cargo c
warning: unused manifest key: target.aarch64-unknown-linux-gnu.linker
   Compiling libc v0.2.119
```
We can probably remove gcc linker from aarch64 target in `Cargo.toml`.

**Comment by Xithrius** [comment]
Ah yes, I forgot to remove that after trying to make aarch64 build in workflows. I'll fix this up.

**Comment by Xithrius** [comment]
Fixed in e3c533ffdc3db76d58e8fad305b7a55f3f4bc3ce
